library/aur.py

```diff
@@ -213,12 +213,7 @@ def main():
                 'default': 'present',
                 'choices': ['present', 'latest'],
             },
-            'ignore_arch': {
-                'default': False,
-                'type': 'bool',
-            },
             'upgrade': {
-                'default': False,
```

Comment:
why are you removing the default here?

Reply:
Ah, so this one was tricky!

If you have a default set for an option, the resulting parameter value for that option is set to whatever the default is, even when the option is omitted in the playbook. This means that the builtin validators which reference this option (like our use of `required_one_of` below) will never cause failures - even when no options are passed at all!

However, if you remove this default then the `upgrade` option is omitted from the parameters, and the validators will correctly flag mis-uses of the API.